packages/astro/src/vite-plugin-astro/hmr.ts

```diff
@@ -58,8 +58,10 @@ export async function handleHotUpdate(
 	}
 }
 
-const frontmatterRE = /^\-\-\-.*?^\-\-\-/ms;
+const frontmatterRE = /^---.+?^---/ms;
```

Comment:
What's different between this regex and `FRONTMATTER_PARSE_REGEXP`?

Reply:
Hmm looks like they're the same. I'll see if they can be re-used